### ISSUE TITLE
Redirect to page besides index

### ISSUE THREAD

**Issue by andrew-teeps** [activeadmin-plugins/active_admin_scoped_collection_actions#14]
I'm trying to do this:

```ruby
scoped_collection_action :email_all do
  me = MassEmail.create(user_ids: scoped_collection_records.pluck(:id), subject: 'Subject here', body: 'Body here')
  redirect_to edit_admin_mass_email_path(me)
end
```

But it seems that the batch action is an ajax call so whatever I render doesn't matter.

Is there any way I can configure this?

**Comment by workgena** [comment]
Unfortunately, In current implementation it is not possible.

Ajax should be rewritten in to form-submit, to allow you do redirects.

https://github.com/activeadmin-plugins/active_admin_scoped_collection_actions/blob/master/vendor/assets/javascripts/active_admin_scoped_collection_actions.js.coffee#L19

Here it make new hash "form_data" with all needed parameters. Instead it we need to add additional parameters to existing form, and action+method attributes to form-tag. Then call .submit() on it instead $.post(). 

Or maybe create dynamic form and submit it.

Today have no time to rewrite this. Perhaps, on the weekend I'll try to do configurable option for this.


Now what I can propose is some ugly solution(with two redirects and session variable). 

```ruby
scoped_collection_action :email_all do
  me = MassEmail.create(user_ids: scoped_collection_records.pluck(:id), subject: 'Subject here', body: 'Body here')
  session[:redirect_after_mass_action] = edit_admin_mass_email_path(me)
  render nothing: true, status: :no_content
end

before_action do
  if session[:redirect_after_mass_action].present?
    url = session[:redirect_after_mass_action] 
    session[:redirect_after_mass_action] = nil
    redirect_to url
  end
end
```

**Comment by andrew-teeps** [comment]
Hey @workgena! I've added my own ajax handler. Thanks!

**Comment by Fivell** [comment]
@andrew-teeps , can you share your code ?

**Comment by Fivell** [comment]
closed by #15
